mpsl/CHANGELOG.rst

```diff
@@ -12,10 +12,16 @@ nRF Connect SDK v2.1.4
 
 All the notable changes included in the |NCS| v2.1.4 release are documented in this section.
 
+Changes
+=======
+
+* On nRF53, now compatible with fix for Errata 160.
```

Comment:
Do we want to keep the same content as in the [1.9.2-dev tag](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.9.2-dev1/nrfxlib/mpsl/CHANGELOG.html#nrf-connect-sdk-v1-9-branch)?
```suggestion
* Removed calls to __WFE() for the nRF5340 SoC, to be in line with the Errata for the Anomaly 160.
```

Reply:
Done!